Sum Root to Leaf Numbers.py

```diff
@@ -1,5 +1,5 @@
 # time complexity is o(n), where n is the size of the input
-# space complexity is o(1)
+# space complexity is o(n), Recursive stack space
```

Comment:
the recursive stack will max contain only height of the tree nodes, it will not go more than that. So here SC is O(h) not O(n)

Reply:
Thank you, I made changes, Thanks for bringing it up.